### ISSUE TITLE
S2 - Text Translation

### ISSUE THREAD

**Issue by nvinogradova** [pace-running/pace3#14]
Business Context:
Currently, when the user is visiting the homepage, the information is provides only in German. In order to provide more accessibility to users with different language proficiencies, the content of the site should be presented in other languages as well, such as: English, Russian, Spanish, Turkish.

Spike objectives:
The goal is to investigate options to translate the content of the website in different languages. Things to consider are pros and cons of each solution and a recommendation based on the given results.
